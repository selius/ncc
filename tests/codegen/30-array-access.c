int main()
{
	int a[5];
	float b[5][4][2];
	int i;
	int j;

	a[3] = 8;
	__print_int(a[3]);

	i = 2;
	j = 1;
	b[1 * i][j * 3][0] = 3.98;
	__print_float(b[1 * i][j * 3][0]);

	return 0;
}
