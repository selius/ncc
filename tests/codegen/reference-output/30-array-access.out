8
3.980000
